SwanCustomEnvironments/swancustomenvironments/scripts/builders/accpy.sh

```diff
@@ -1,24 +1,31 @@
 #!/bin/bash
 
+# Create a middle layer for installing Spark extensions, putting them apart from the user environment
+uv venv $SWAN_ENV --seed 2>&1
+source $SWAN_ENV/bin/activate
+uv pip install "ipykernel==${IPYKERNEL_VERSION}"
+SWAN_PACKAGES_PATH=$(python3 -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])')
+deactivate
+
 # If using NXCALS, we need to install the Spark extensions and the nxcals package.
```

Comment:
If we don't install nxcals anymore, the variable INSTALL_NXCALS should change its name?